lib/recurly/base.php

```diff
@@ -227,99 +199,37 @@ public function getLinks() {
     return $this->_links;
   }
 
+  public function getErrors()
+  {
+    return $this->_errors;
+  }
+
+  public function resetErrors()
+  {
+    $this->_errors = array();
+  }
+
+  // depends on $this->getNodeName()
+  protected function updateErrorAttributes()
+  {
+    if (sizeof($this->_errors) > 0) {
+      for ($i = sizeof($this->_errors) - 1; $i >= 0; $i--) {
```

Comment:
maybe $i could be $errorIndex?

Reply:
If I'm reading this right, `$i` represents the size of that array, yea? This was just a method I had shuffled around so I didn't actually write this code.. 😅